dahee/app.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express')
+const cors = require('cors')
+const app = express()
+const dotenv = require('dotenv')
+
+dotenv.config()
+
+app.use(cors())
+app.use(express.json())
+//app.use(morgan("combined"))
```

Comment:
```suggestion
app.use(morgan("combined"))
```
- 해당 middleware를 사용하는 것이라면 주석을 지워주시고, 사용하지 않는다고 하면 코드를 지워주세요.
- 코드가 주석처리된 채로 남아있다면 필요한 상황에 제대로 사용되거나 삭제되는 일은 거의 없고 결국 계속 남아있게 됩니다.
- 그러다보면 이제 작성한 사람이 아닌 다른 개발자는 필요한 코드라서 주석처리한게 아닐까? 라는 생각으로 지우지 못하고 결국 그 누구도 왜 있는지는 모르면서 계속 남아있는 불필요한 레거시가 되어버립니다!
- 따라서 반드시 필요한 경우가 아니면 코드를 주석처리하기보다는 삭제해주시고 만약 다시 보고싶다면 커밋 이력에 잘 남겨뒀다가 원하는 시점에 다시 보면 됩니다!